src/include/stir/IO/InputStreamFromROOTFileForCylindricalPET.h

```diff
@@ -1,10 +1,3 @@
-/*!
```

Comment:
need to keep this. this is the doc for the file, which is spearate for the one for the class